aci/resource_aci_fvaepg.go

```diff
@@ -173,11 +173,51 @@ func resourceAciApplicationEPG() *schema.Resource {
 				Optional: true,
 				Set:      schema.HashString,
 			},
-			"relation_fv_rs_node_att": &schema.Schema{
+			"relation_fv_rs_node_att": &schema.Schema{ 
```

Comment:
should we look at state upgraders for this change or is hard change like this sufficient? See example link: https://github.com/CiscoDevNet/terraform-provider-aci/pull/1040/files#diff-36dce9822415b973aec1985a3676542ea9e320dbe3145ef5bf8a52d094202030R75

Reply:
It is not working properly today so I don't see the need for a state upgrader. What do you think?

Reply:
Ok that makes sense leave as is then.